### PR TITLE
Task-50955 : Add Points field & Program field to challenge drawer

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**
add points field and program filed  to challenge drawer.

### CODE REVIEW

**Review by hbenali — COMMENTED** [review]
typo

File: challenges-webapp/src/main/resources/locale/portlet/Challenges_en.properties [hunk 1]
```diff
@@ -66,3 +66,6 @@ challenges.announcementCreateSuccess=Announcement created successfully.
 challenges.permissionDenied=You don\u2019t have the permission to announce
 challenges.challengeNotStarted=The challenge did not start yet
 challenges.challengeEnded=The challenge has ended
+challenges.label.points=Points
+challenges.label.reward=Reward
+challenges.label.pointsValidation=the number of points must be between  0 and 9999
```
Comment (hbenali) [inline challenges-webapp/src/main/resources/locale/portlet/Challenges_en.properties]:
```suggestion
challenges.label.pointsValidation=The number of points must be between 0 and 9999
```

Comment (Rdidmohamedelhadi) [reply challenges-webapp/src/main/resources/locale/portlet/Challenges_en.properties]: Thanks @hbenali :man_technologist:

**Comment by ahamdi** [comment]
@Rdidmohamedelhadi Please update description

**Review by ahamdi — APPROVED** [review]

**Pull request merged**